src/Hypernode/Magento/Command/Hypernode/Modules/ListUpdatesCommand.php

```diff
@@ -65,42 +68,52 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return;
         }
 
-        $modules = $this->getModules()
-                ->filterModules($input);
+        $onlyActive = $input->getOption('only-active') ? true : false;
```

Comment:
Yes you can do that. My IDE suggested to typecast it as it it's approx. 6x times faster. 

![screen shot 2016-12-23 at 17 05 02](https://cloud.githubusercontent.com/assets/4974910/21457772/07d680d6-c932-11e6-9f38-69944e029e1c.png)

But we don't have to set this variable at all. `InputOption::VALUE_NONE` is already a true / false construction by default.

Because this variable is only used twice, we could skip setting it completely. The option name is descriptive enough so i removed these lines.